tests/py/test_collection.py

```diff
@@ -0,0 +1,110 @@
+import torch
```

Comment:
Put this in hub.py so it will get created with all the other test models. Also make the names more descriptive so its clear they focus on collections.